modin/core/storage_formats/base/query_compiler.py

```diff
@@ -2099,13 +2110,22 @@ def apply(self, func, axis, *args, **kwargs):
             QueryCompiler that contains the results of execution and is built by
             the following rules:
 
-            - Labels of specified axis are the passed functions names.
+            - Index of the specified axis contains: the names of the passed functions if multiple
+              functions were passed, otherwise: indices of the `func` result if "expand" strategy
+              was used, indices of the original frame if "broadcast" strategy was used, a single
+              label "__reduced__" if "reduce" strategy was used.
```

Comment:
```suggestion
            - Index of the specified axis contains: the names of the passed functions if multiple
              functions are passed, otherwise: indices of the `func` result if "expand" strategy
              is used, indices of the original frame if "broadcast" strategy is used, a single
              label "__reduced__" if "reduce" strategy is used.
```